src/graphinfo.jl

```diff
@@ -217,6 +218,11 @@ function Base.getindex(m::Model, vn::VarName)
     return m.g[vn]
```

Comment:
Do we need the following for dereferencing variables? 
```suggestion
    return m.g[vn][]
```

Reply:
I added a `setvalue!` function and got rid of the `setindex`.